src/donejs-carousel-plugin.js

```diff
@@ -396,20 +419,22 @@ export const ViewModel = DefineMap.extend({
 	*
 	*/
 	swipeEnd() {
+		let touchEvent = this.defineTouchEvent(event);
```

Comment:
that's weird. wasn't this code always in the repo? not sure why it says it's being added here.

Reply:
this code was in the swipeMove function, added it also in the swipeEnd because it should only be executing fully if dragging and touch conditions are met.